core/experiments/assets/js/admin/behaviors/experiments-messages.js

```diff
@@ -58,6 +65,17 @@ export default class ExperimentsDependency {
 			) );
 	}
 
+	getExperimentDeactivatingDialogMessage( experimentId ) {
```

Comment:
```suggestion
	getMessage( experimentId, messageId ) {
```

Reply:
What do you refer by `messageId`what should I pass?

Reply:
Done